src/components/codeBlock.tsx

```diff
@@ -106,52 +106,154 @@ type KeywordSelectorProps = {
 };
 
 function OrgAuthTokenCreator() {
-  const codeContext = useContext(CodeContext);
+  const [tokenState, setTokenState] = useState<
+    | {status: 'none'}
+    | {status: 'loading'}
+    | {status: 'success'; token: string}
+    | {status: 'error'}
+  >({status: 'none'});
+  const [isOpen, setIsOpen] = useState(false);
+  const [referenceEl, setReferenceEl] = useState<HTMLSpanElement>(null);
+  const [dropdownEl, setDropdownEl] = useState<HTMLElement>(null);
+  const {codeKeywords} = useContext(CodeContext);
```

Comment:
nit: I would move the context subscription to the first line then it is easier to spot.
(as it is a dependency that component has)